src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -189,6 +192,16 @@ private static WebinCliParameters initParameters(String submissionAccount, Webin
         }
     }
 
+    private void validateParameters() {

```

Comment:
- Please remove validateParameters. It is not needed when using WebinSubmissionTool (EnaSubmissionTool) enum in WebinCliParameters.